core/src/scene/sceneLoader.cpp

```diff
@@ -1049,6 +1024,69 @@ std::vector<StyleParam> SceneLoader::parseStyleParams(Node params, Scene& scene,
     return out;
 }
 
+StyleUniforms SceneLoader::parseStyleUniforms(const Node& value, Scene& scene) {
+    std::string type = "";
+    std::vector<UniformValue> uniformValues;
+    if (value.IsScalar()) { //float, int (bool), string (texture)
+        UniformValue uVal;
+        try {
+            uVal = value.as<float>();
+            type = "float";
+        } catch (const BadConversion& e) {
+            auto strVal = value.as<std::string>();
+            type = "int";
```

Comment:
Not just feature parity.

But it also gets harder to distinguish between an `int` or `float` value in yaml. Its doable though!!!! But!!!

With respect usage of `int` uniforms. Based on a quick discussion we had here in office, it seems int uniforms are mostly used when you want to access array indices. We could not come up with any other use case for `int` uniforms.